pom.xml

```diff
@@ -30,6 +30,8 @@ limitations under the License.
     <packaging>pom</packaging>
 
     <properties>
+        <artifactory-build-name>thirdparty-bom</artifactory-build-name>
```

Comment:
Can this not be sourced from the project artifactId?

Reply:
This is a "build name", the one name to rule all the artifacts produced by this project - it names the build-info entry in artfactory.   If I set it to project.artifactId, it might (not tested) produce unexpected results.   

The other option I can think of is to externalize that into a required CI environment var or system property.  I don't expect it to ever change, hence the above.